diff -u ../editors/linux-sublime3/Makefile editors/linux-sublime3/Makefile
--- editors/linux-sublime3/Makefile	2021-05-07 17:44:30.878400000 +0200
+++ editors/linux-sublime3/Makefile	2021-05-22 18:31:54.534857000 +0200
@@ -2,23 +2,33 @@
 
 PORTNAME=	sublime
 PORTVERSION=	3.2.2
+PORTREVISION=	1
 CATEGORIES=	editors linux
 MASTER_SITES=	http://download.sublimetext.com/
 PKGNAMEPREFIX=	linux-
 PKGNAMESUFFIX=	3
-DISTNAME_amd64=	sublime_text_3_build_3211_x64
-DISTNAME_i386=	sublime_text_3_build_3211_x32
 
-MAINTAINER=	ports@FreeBSD.org
+MAINTAINER=	matias@pizarro.net
 COMMENT=	Sophisticated text editor for code, markup and prose
 
+LICENSE=				SUBLIMEHQ
+LICENSE_NAME=			End User License Agreement
+LICENSE_FILE=			${FILESDIR}/LICENSE.txt
+LICENSE_PERMS=			dist-mirror no-dist-sell pkg-mirror no-pkg-sell auto-accept
+
+ONLY_FOR_ARCHS=			amd64 i386
+ONLY_FOR_ARCHS_REASON=	Upstream only supports amd64 and i386
+
 USES=		desktop-file-utils gnome linux tar:bzip2
 USE_LINUX=	gtk3
 
+DISTNAME_amd64=	sublime_text_3_build_3211_x64
+DISTNAME_i386=	sublime_text_3_build_3211_x32
+
 INSTALLS_ICONS=	yes
 WRKSRC=		${WRKDIR}/sublime_text_3
 
-CONFLICTS_INSTALL=	linux-sublime-[0-9]*
+CONFLICTS_INSTALL=	linux-sublime[0-9]*
 
 NO_BUILD=	yes
 SUB_FILES=	sublime
Common subdirectories: ../editors/linux-sublime3/files and editors/linux-sublime3/files
